### ISSUE TITLE
How to draw chart's frame always

### ISSUE THREAD

**Issue by Yukimune** [beto-rodriguez/Live-Charts#304]
Hi, I'm trying to draw frame of chart always.
Now:
![1](https://cloud.githubusercontent.com/assets/5716130/17800142/8dfaebb8-661b-11e6-8d73-c3e968bb6ee8.png)
Desire:
![2](https://cloud.githubusercontent.com/assets/5716130/17800163/b64687f8-661b-11e6-8aa7-1bf6ca113df5.png)

I think it's possible by setting maximumvalue(binding oneway) to Axis.MaxValue , AxisSection.From,  and AxisSection.To. 
Is there any better ideas?
Thank you.

**Comment by beto-rodriguez** [comment]
By default each series according to its type returns its range to be drawn correctly, as I can see here it is doing it wrong, can you please let me know your values and chart's height so I can force the issue and improve this default behavior?

**Comment by beto-rodriguez** [comment]
This was wrong, last commit gives more space by default to a Line series and Vertical Line series

**Comment by Yukimune** [comment]
Thank you so much!!
I've tried last commit. It's cool and just I wanted!;)

**Comment by Yukimune** [comment]
I thank for your sincere work.
After tried some cases, I found a case that doesn't draw a separator on top of the chart.
My goal is all points are plotted inside of separator.

In this case, y:30.9 is the all series' max separator.  But this sample doesn't like to be expected.
Here is sample code.(using library is a commit "fixes #304")

MainWindow.xaml.cs
```
    public partial class MainWindow : Window
    {
        public SeriesCollection Lines { get; } = new SeriesCollection();

        public MainWindow()
        {
            InitializeComponent();

            Lines.Add(new LineSeries()
            {
                Values = new ChartValues<double>()
                {
                    30.6257680042767,
                    30.8093094266063,
                    30.8506761248459
                }
            });

            DataContext = this;
        }
    }
```

MainWindow.xaml
```
<Window x:Class="WpfApplication1.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:lvc="clr-namespace:LiveCharts.Wpf;assembly=LiveCharts.Wpf"
        xmlns:local="clr-namespace:WpfApplication1"
        mc:Ignorable="d"
        d:DataContext="{d:DesignInstance Type=local:MainWindow}"
        Title="MainWindow" Height="350" Width="525">
    <Grid>
        <lvc:CartesianChart Series="{Binding Lines}" >
            <lvc:CartesianChart.AxisX>
                <lvc:Axis MaxValue="10"></lvc:Axis>
            </lvc:CartesianChart.AxisX>
        </lvc:CartesianChart>
    </Grid>
</Window>

```
Fact
![1](https://cloud.githubusercontent.com/assets/5716130/17954543/a07d07c2-6ab6-11e6-8928-5c23a55dd875.png)


Expected
![2](https://cloud.githubusercontent.com/assets/5716130/17954529/954b401c-6ab6-11e6-97a4-ac052586c62a.png)

Thank you!;)
